packages/picasso.js/src/core/chart-components/ref-line/lines-and-labels.js

```diff
@@ -225,8 +323,81 @@ export function createLineWithLabel({ chart, blueprint, renderer, p, settings, i
   // Always push the line,
   // but this is done after collision detection,
   // because otherwise it would collide with it's own line
-  items.push(line);
+  if (line) {
+    items.push(line);
+  } else if (slopeLine) {
+    // push slope line
+    items.push(slope);
+    // create data area labels for slope line
+    let valueString;
+    let rectLabel;
```

Comment:
Maybe rename this variable to `labelBackground` or `labelRect` to make it more clear what it is?

Reply:
ok can change it to labelRect then ...